iml-wire-types/src/lib.rs

```diff
@@ -1590,16 +1590,62 @@ impl PacemakerOperations {
             stop: stop.into(),
         }
     }
+
+    pub fn is_some(&self) -> bool {
```

Comment:
This name is confusing as it makes it appear that entire struct is wrapped in an Option.